### ISSUE TITLE
Repeat/Shuffle modes?

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#71]
```
I would like to see an option to toggle between repeat/shuffle/normal playing modes.
 Currently, I don't see that in any of the menus or options.

I'm using version 0.9 on Android.
```

Original issue reported on code.google.com by `mk73183` on 2013-03-24 05:36:35

**Comment by kaaholst** [comment]
```
Your request is hereby noted, and for your information, it is already being worked on
in a development branch of the Squeezer source code.

```

Original issue reported on code.google.com by `kurt.aaholst@lumigon.com` on 2013-03-25 10:01:29

**Comment by kaaholst** [comment]
```
Hi,

I just joined the "beta" community but am not yet able to install the beta version.
(Might take some time through Google Play.)

Can you specify if shuffle / repeat is already available in the 1.0 beta?

Cheers!
```

Original issue reported on code.google.com by `gijsmob` on 2013-12-11 19:20:12

**Comment by kaaholst** [comment]
```
Yes shuffle and repeat are available in 1.0 beta.

I don't know if you have done that already, but after joining you need to opt in to
be able to install the beta version, by following this link, and follow the instructions.
https://play.google.com/apps/testing/uk.org.ngo.squeezer
This link can also be found on the Squeezer Beta Testers page.
```

Original issue reported on code.google.com by `kaaholst` on 2013-12-11 20:33:04

**Comment by kaaholst** [comment]
```
1.0.0 has now been released, which has support for both repeat and shuffle.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2013-12-18 21:19:05
